test/userController.test.js

```diff
@@ -58,9 +58,28 @@ afterAll(async () => {
   await mongodb.stop();
 });
 
+beforeEach(async () => {
+  // Delete all users with the same email before creating the test user
+  await User.deleteMany({ email: testUserCredentials.email });
+
+  // Create a test user before each test
+  testUser = await User.create({
+    name: 'ava',
+    username: 'ava',
+    email: 'ava@ava.com',
+    password: 'secret',
+    role: 'user',
+  });
+});
+
+afterEach(async () => {
+  // Delete the test user after each test
+  await User.deleteMany({});
```

Comment:
Since we already do a `deleteMany` specifically for the testUser in the `beforeEach`, why not just do this `User.deleteMany({})` in the beforeEach instead? Since it's already deleting all users

Reply:
<img width="691" alt="jest" src="https://github.com/Code-the-Dream-School/dd-prac-team4-back/assets/95896159/a1b94ed1-0463-49cc-8da4-ee7d8f2f7160">